sources/Adapters/picoTracker/display/font_wide.h

```diff
@@ -0,0 +1,108 @@
+    {0x000, 0x0FC, 0x186, 0x186, 0x186, 0x1FC, 0x180, 0x180, 0x0FC, 0x000}, // 9
+    {0x000, 0x000, 0x000, 0x018, 0x018, 0x000, 0x018, 0x018, 0x000, 0x000}, // :
+    {0x000, 0x000, 0x000, 0x018, 0x018, 0x000, 0x018, 0x018, 0x00C, 0x000}, // ;
+    {0x000, 0x180, 0x0C0, 0x060, 0x030, 0x030, 0x060, 0x0C0, 0x180, 0x000}, // <
+    {0x000, 0x000, 0x000, 0x1FC, 0x000, 0x000, 0x1FC, 0x000, 0x000, 0x000}, // =
+    {0x000, 0x006, 0x00C, 0x018, 0x030, 0x030, 0x018, 0x00C, 0x006, 0x000}, // >
+    {0x000, 0x0FC, 0x186, 0x180, 0x0C0, 0x060, 0x060, 0x000, 0x060, 0x000}, // ?
+    {0x000, 0x0FC, 0x186, 0x1B6, 0x196, 0x196, 0x1F6, 0x006, 0x0FC, 0x000}, // @
+    {0x000, 0x0FC, 0x186, 0x186, 0x186, 0x1FE, 0x186, 0x186, 0x186, 0x000}, // A
+    {0x000, 0x0FE, 0x186, 0x186, 0x0FE, 0x186, 0x186, 0x186, 0x0FE, 0x000}, // B
+    {0x000, 0x0FC, 0x186, 0x006, 0x006, 0x006, 0x006, 0x186, 0x0FC, 0x000}, // C
+    {0x000, 0x0FE, 0x186, 0x186, 0x186, 0x186, 0x186, 0x186, 0x0FE, 0x000}, // D
+    {0x000, 0x1FE, 0x006, 0x006, 0x07E, 0x006, 0x006, 0x006, 0x1FE, 0x000}, // E
+    {0x000, 0x1FE, 0x006, 0x006, 0x07E, 0x006, 0x006, 0x006, 0x006, 0x000}, // F
+    {0x000, 0x0FC, 0x186, 0x006, 0x006, 0x1E6, 0x186, 0x186, 0x0FC, 0x000}, // G
+    {0x000, 0x186, 0x186, 0x186, 0x1FE, 0x186, 0x186, 0x186, 0x186, 0x000}, // H
+    {0x000, 0x1FE, 0x030, 0x030, 0x030, 0x030, 0x030, 0x030, 0x1FE, 0x000}, // I
+    {0x000, 0x1E0, 0x0C0, 0x0C0, 0x0C0, 0x0C0, 0x0C0, 0x0E6, 0x07C, 0x000}, // J
+    {0x000, 0x186, 0x0C6, 0x066, 0x03E, 0x03E, 0x066, 0x0C6, 0x186, 0x000}, // K
+    {0x000, 0x006, 0x006, 0x006, 0x006, 0x006, 0x006, 0x006, 0x1FE, 0x000}, // L
+    {0x000, 0x186, 0x1CE, 0x1FE, 0x1B6, 0x186, 0x186, 0x186, 0x186, 0x000}, // M
+    {0x000, 0x186, 0x18E, 0x19E, 0x1BE, 0x1F6, 0x1E6, 0x1C6, 0x186, 0x000}, // N
+    {0x000, 0x0FC, 0x186, 0x186, 0x186, 0x186, 0x186, 0x186, 0x0FC, 0x000}, // O
+    {0x000, 0x0FE, 0x186, 0x186, 0x186, 0x0FE, 0x006, 0x006, 0x006, 0x000}, // P
+    {0x000, 0x0FC, 0x186, 0x186, 0x186, 0x186, 0x136, 0x066, 0x0DC, 0x000}, // Q
+    {0x000, 0x0FE, 0x186, 0x186, 0x186, 0x0FE, 0x186, 0x186, 0x186, 0x000}, // R
+    {0x000, 0x0FC, 0x186, 0x006, 0x0FC, 0x180, 0x180, 0x186, 0x0FC, 0x000}, // S
+    {0x000, 0x1FE, 0x030, 0x030, 0x030, 0x030, 0x030, 0x030, 0x030, 0x000}, // T
+    {0x000, 0x186, 0x186, 0x186, 0x186, 0x186, 0x186, 0x186, 0x0FC, 0x000}, // U
+    {0x000, 0x186, 0x186, 0x186, 0x186, 0x186, 0x0CC, 0x078, 0x030, 0x000}, // V
+    {0x000, 0x186, 0x186, 0x186, 0x186, 0x1B6, 0x1FE, 0x1CE, 0x186, 0x000}, // W
+    {0x000, 0x186, 0x186, 0x0CC, 0x078, 0x0CC, 0x186, 0x186, 0x186, 0x000}, // X
+    {0x000, 0x186, 0x186, 0x186, 0x0CC, 0x078, 0x030, 0x030, 0x030, 0x000}, // Y
+    {0x000, 0x1FE, 0x180, 0x0C0, 0x070, 0x018, 0x00C, 0x006, 0x1FE, 0x000}, // Z
+    {0x000, 0x078, 0x018, 0x018, 0x018, 0x018, 0x018, 0x018, 0x078, 0x000}, // [
+    {0x000, 0x002, 0x006, 0x00C, 0x018, 0x030, 0x060, 0x0C0, 0x180,
+     0x000}, // backslash
```

Comment:
small nitpick, could you use the same `/* \*/` as in the other font definitions please